recipe_scrapers/chefkoch.py

```diff
@@ -0,0 +1,28 @@
+from ._abstract import AbstractScraper
+
+
+class Chefkoch(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"chefkoch.{domain}"
```

Comment:
While I'm not opposed to using the `kwarg`, we do this when there are multiple domains to the same website. Take a look at [this example](https://github.com/hhursev/recipe-scrapers/blob/60ff6a52ce81eafae79b27c4775474ce3b979fea/recipe_scrapers/__init__.py#L103).

Is there another domain that this website is available at?

Reply:
There is none. It's just "chefkoch.de". Do you want me to remove the `domain` parameter?

Reply:
Yes, I'd prefer that we maintain the style of the package and only use that pattern when there are multiple TLDs possible.